src/zino/api/legacy.py

```diff
@@ -233,8 +234,11 @@ async def do_version(self):
     @requires_authentication
     async def do_caseids(self):
         self._respond(304, "list of active cases follows, terminated with '.'")
-        for event_id in sorted(self._state.events.events):
-            self._respond_raw(str(event_id))
+        events = self._state.events.events
+        event_ids = sorted(events)
+        for event_id in event_ids:
+            if events[event_id].state is not EventState.CLOSED:
+                self._respond_raw(str(event_id))
```

Comment:
```suggestion
        events = self._state.events.events
        for event_id, event in sorted(events.items()):
            if event.state != EventState.CLOSED:
                self._respond_raw(str(event_id))
```

Reply:
Alternatively, the `Events` class could gain methods that do the filtering on closed/non-closed states - but let's do that only if we see that we need this in more places.